src/backend-api/Controllers/AddController.cs

```diff
@@ -519,7 +495,11 @@ public IActionResult PostProgram([FromBody] PostProgramInputModel input)
         [Route("Plugin")]
         public IActionResult PostPlugin([FromBody] PostPluginInputModel input)
         {
-            if (!(_context.Program.Select(x => x.ProgramName).ToList().Contains(input.ProgramName)))
+            // this is checking to see if the program exists that the input claims it's a plugin of
+            if (!(_context.Program
+                .Select(x => x.ProgramName)
+                .ToList()
+                .Contains(input.ProgramName)))
```

Comment:
Perhaps put the return statement in curly braces to show what is the result of the if statement.